lambdatimes/src/components/Content/Card.js

```diff
@@ -1,19 +1,27 @@
-import React from 'react';
+import React, {Component} from 'react';
+import PropTypes from "prop-types"
 
 const Card = props => {
   return (
     <div className="card">
-      <div className="headline">{/* headline goes here */}</div>
+      <div className="headline">{props.headline}</div>
       <div className="author">
         <div className="img-container">
-          <img src={'' /* image source goes here */} />
+          <img src={props.img} alt="image" />
         </div>
-        <span>By {/* author goes here */}</span>
+        <span>By {props.author}</span>
```

Comment:
You also have access to all the info on the card prop so you could have accessed it via props.card.author props.card.headline etc